src/client/components/Meals.js

```diff
@@ -0,0 +1,29 @@
+
+import React, {useEffect, useState,createContext} from "react";
+import PropTypes from "prop-types";
+import { DisplayMeals } from "./DisplayMeals";
+import { Link } from "react-router-dom";
+
+export default function Meals() {
+  const [title, setNewTitle] = useState([]);
```

Comment:
This is a better naming. it is not titles but meals state.

Needs to be renamed in the rest of the file too
```suggestion
  const [meals, setMeals] = useState([]);
```